packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1161,8 +1161,43 @@ The Multifactor (MFA) Enrollment Policy controls which MFA methods are available
    }
 ```
 
+#### Policy Authenticators Settings example
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+> **Note:** Policy Settings are included only for those Authenticators that are enabled.
+
+```json
+   "settings": {
+     "type": "AUTHENTICATORS",
+     "authenticators": [
+       {
+         "key": "security_question",
+         "enroll": {
+           "self": "OPTIONAL"
+         }
+       },
+       {
+         "key": "okta_phone",
+         "enroll": {
+           "self": "OPTIONAL"
+         }
+       }
+     ]
+   }
+```
+
 ### Policy Settings data
 
+> **Note:** Default type is `FACTORS` when not specified.
```

Comment:
Instead of this, just add a `Default` column to the table like we do in other parameter tables

![image](https://user-images.githubusercontent.com/33136962/134256016-b3a23e9a-fcc3-4d2b-9b15-c313465036f3.png)